package.json

```diff
@@ -49,11 +49,9 @@
     "build": "ng build",
     "e2e": "ng build --no-progress && concurrently \"npm run json-server\" \"ng e2e --proxy-config config/proxy.conf.json\" --kill-others --success first",
     "e2e-no-server": "ng e2e --no-serve --proxy-config config/proxy.conf.json",
-    "e2e-report": "npm run e2e-no-server && open ./e2e/reports/html/feature_test_report.html",
-    "e2e-report-open": "open ./e2e/reports/html/feature_test_report.html",
     "hmr": "ng serve --open --hmr -e=hmr --proxy-config config/proxy.conf.json",
     "json-server": "nodemon --watch ./json-server/ ./json-server/server.js --delay 2",
-    "lint": "ng lint --type-check",
+    "lint": "ng lint",
```

Comment:
do we want to add --fix here to auto fix lint errors (at least the ones that can be)?